demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1810,23 +1856,25 @@ export class DemoMeetingApp
         new DefaultEventController(configuration, this.meetingLogger, this.eventReporter)
     );
 
+    const enableAudioRedundancy = !((document.getElementById('disable-audio-redundancy') as HTMLInputElement).checked);
+    this.log(`Audio Redundancy Enabled = ${enableAudioRedundancy}`);
```

Comment:
nit: Should this log line be emitted after the audio profile is set, since this log line will suggest that RED is enabled even for Firefox clients?
```
this.log(`Audio Redundancy Enabled = ${audioProfile.hasRedundancyEnabled}`);
```

Reply:
Updated